README.md

```diff
@@ -1 +1,80 @@
-# capstone1
+## Description of the project
```

Comment:
### Please add a Title for this project. 👍🏻

Reply:
- Kindly make sure this is done 👍🏽